tests/test_F0_output.py

```diff
@@ -0,0 +1,189 @@
+"""
+- Aggregate demand profiles to a total demand profile
+- Plotting all energy flows for both 14 and 365 days for each energy bus
+- Store timeseries of all energy flows to excel (one sheet = one energy bus)
+- execute function: plot optimised capacities as a barchart (F1)
+- execute function: plot all annuities as a barchart (F1)
+- store scalars/KPI to excel
+- process dictionary so that it can be stored to Json
+- store dictionary to Json
+"""
+
+import os
+import sys
+import shutil
+import pytest
+import argparse
+import logging
+
+import pandas as pd
+import numpy as np
+import src.F0_output as F0
+
+OUTPUT_PATH = os.path.abspath(os.path.join(".", "tests", "test_outputs"))
+
+start_time = "2020-01-01 00:00"
+periods = 3
+values = [0, 1, 2]
+
+pandas_DatetimeIndex = pd.date_range(start=start_time, periods=periods, freq="60min")
+pandas_Series = pd.Series(values, index=pandas_DatetimeIndex)
+pandas_Dataframe = pd.DataFrame({"a": values, "b": values})
+scalar = 2
+
+json_test_dictionary = {
```

Comment:
if those are constants and not variables, I would capitalize them as recommended by [PEP8](https://www.python.org/dev/peps/pep-0008/#constants)

Reply:
Done